src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentalPrice = 40;
+  const rentalAmount = days * rentalPrice;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+
+  if (days < 3) {
+    return rentalAmount;
+  }
+
+  if (days < 7) {
+    return rentalAmount - threeDayDiscount;
+  }
+
+  if (days >= 7) {
+    return rentalAmount - sevenDayDiscount;
+  }
```

Comment:
this if is implicit - if other two fail, this will apply, so no need for an explicit `if`, just `return`:
```suggestion
    return rentalAmount - sevenDayDiscount;
```

Reply:
Done!